turbinia/evidence.py

```diff
@@ -470,8 +495,8 @@ class PlasoFile(Evidence):
   def __init__(self, plaso_version=None, *args, **kwargs):
     """Initialization for Plaso File evidence."""
     self.plaso_version = plaso_version
-    self.copyable = True
-    super(PlasoFile, self).__init__(*args, **kwargs)
+    super(PlasoFile, self).__init__(copyable=True, *args, **kwargs)
```

Comment:
I think this is fine, but why do we need to pass this into the constructor rather than just setting it after the constructor runs?

Reply:
Because I check that source_path is set if copyable is True in the Evidence constructor

Reply:
Interesting. This could be pretty easy for people to miss when creating future evidence types that are copyable, and then the `source_path` check wouldn't happen (and other weird failures would happen down the line potentially).   I can't really think of a good way to fix that though.  We could add a `hasattr()` check in the parent constructor, but that seems pretty hacky.  Maybe we should put a comment somewhere at least.

Reply:
Setting copyable to True after super() would work.
The only risk is if you also forget to set the source_path when instantiating a new CopyableEvidence in a Task (which you should. And current tasks that generate "copyable" evidences objects do that). My check in Evidence.__init__() won't trigger and you may end up with errors when the framework tries to move your data